scene/resources/texture.cpp

```diff
@@ -438,12 +449,17 @@ void PortableCompressedTexture2D::create_from_image(const Ref<Image> &p_image, C
 	switch (p_compression_mode) {
 		case COMPRESSION_MODE_LOSSLESS:
 		case COMPRESSION_MODE_LOSSY: {
+			bool lossless_force_png = GLOBAL_GET("rendering/textures/lossless_compression/force_png") ||
+					!Image::_webp_mem_loader_func; // WebP module disabled.
+			bool use_webp = !lossless_force_png && p_image->get_width() <= 16383 && p_image->get_height() <= 16383; // WebP has a size limit
```

Comment:
```suggestion
			bool use_webp = !lossless_force_png && p_image->get_width() <= 16383 && p_image->get_height() <= 16383; // WebP has a size limit.
```
Comment format nitpick

Reply:
Ok) this is just a copy from [`resource_importer_texture.cpp`](https://github.com/godotengine/godot/blob/95c8216923fc9353ac50f469836ae149cb196e83/editor/import/resource_importer_texture.cpp#L247)